src/Uno.UI.Runtime.Skia.Android/Microsoft/UI/Xaml/ApplicationActivity.cs

```diff
@@ -448,10 +448,7 @@ public SKPath Path
 			protected override void OnDraw(Canvas canvas)
 			{
 				base.OnDraw(canvas);
-				if (!string.IsNullOrEmpty(_svgClipPath))
```

Comment:
Shouldn't such change be needed on other targets as well? Because originally the issue was found on iOS, so I think this fixes just the Android part